### ISSUE TITLE
Include zIndex data when exporting canvas to json

### ISSUE THREAD

**Issue by ehussain** [pikasojs/pikaso#63]
**Is your feature request related to a problem? Please describe.**
I am trying to use certain method for setting zIndex of shape. 
Currently this feature is not available on pikaso, however Konva has that ability to set zIndex.
https://konvajs.org/docs/groups_and_layers/zIndex.html

Problem started when I created one editor with Image and Text on top of that. Later I exported JSON and when I import that JSON back into editor then sequence or zindex is lost, image always comes on top and text on bottom hiding behind it. Having an option to set specific z index will solve the issue.

**Describe the solution you'd like**
If there is a way to access the native Konva shape method then it could solve lot of issues.

**Describe alternatives you've considered**
If it is not possible then there should be  specific way to set index of object. 
Currently shape.node has index property however there is no way to change the index of objects if we want to move one on top of other.

**Comment by raminious** [comment]
Hi @ehussain 

There are two ways to access `Konva` shapes through `Pikaso`.  
An example will help me to explain 

```ts
const circle = editor.shapes.circle.insert({ 
  /* config */
})
```

here `circle.node` has access to the [Konva](https://pikaso.app/api/classes/CircleModel.html#node) shape.
So in your case `circle.node.zIndex(1)` is the solution.  

The second way is using [update](https://pikaso.app/api/classes/CircleModel.html#update) method that is provided by Pikaso. 

```ts
circle.update({
  zIndex: 1
})
```

**Comment by ehussain** [comment]
Hello @raminious 

Thanks for quick response. This is very helpful. I was able to adjust zindex of shapes using shape.update({zIndex: 1}) method as you have mentioned.

However I am still facing the issue in following scenario.

1. let imageShape = editor.shapes.image.insert(ImageURL) - Inserts Image URL 
2. let labelShape = editor.shapes.label.insert({}) - Inserts "Untitled Text"
3. labelShape.update({zIndex : 99}) // making sure label remains on top and it shows properly
4. let json = editor.export.toJson();
5. editor.reset() // reset canvas
6. editor.import.json(json) // load previous json again

After doing above steps there is issue where image always takes top place and label gets hidden behind image. 

Thanks in advance.

**Comment by raminious** [comment]
Ok, let me try to reproduce the scenario

**Comment by raminious** [comment]
@ehussain I could reproduce the bug [https://codesandbox.io/s/floral-field-o4op8c?file=/src/index.ts](https://codesandbox.io/s/floral-field-o4op8c?file=/src/index.ts)

Pikaso relies on Konva's `toObject` method when exporting the board to JSON. in this case Konva doesn't return `zIndex` with created object of `toObject`  
This is a Konva bug, and I think they should fix it, but I should fix it myself since Konva does not support exporting to JSON.

**Comment by ehussain** [comment]
Thanks.

**Comment by raminious** [comment]
@ehussain 
The issue has been fixed and with v2.6.0 will be released this week. 
I appreciate your contribution.

**Comment by raminious** [comment]
released https://github.com/pikasojs/pikaso/releases/tag/v2.6.0
